nanome_vault/Vault.py

```diff
@@ -37,12 +37,16 @@ def on_run(self):
         self.menu.show_menu()
 
     @async_callback
-    async def on_export_file(self, request):
+    async def on_export_integration(self, request):
         await self.on_presenter_change()
         (location, filename, data) = request.get_args()
-        path = os.path.join(self.account, location)
-        self.vault.add_file(path, filename, data)
-        request.send_response(True)
+
+        if location == 'Browse':
```

Comment:
Would it be a lot of work to make EXPORT_LOCATIONS into an enum? I feel like this should look like

```
if location == EXPORT_LOCATIONS.BROWSE:
    ...
```